site/dat/repo/various.json

```diff
@@ -636,6 +636,11 @@
       "com.ubikingenierie.jmeter.plugin.hls.gui.HLSSamplerGUI"
     ],
     "versions": {
+    "6.6.0": {
```

Comment:
Can I ask you why the PR included changes on another plugin ?  Please provide a clean PR

Reply:
Hello! I thought it was weird too but, the version that it's now in master contains those lines as well (I just copy-pasted it tbh).
As can be seen here: https://github.com/undera/jmeter-plugins/blob/master/site/dat/repo/various.json